prow/README.md

```diff
@@ -155,3 +155,28 @@ $ chmod +x prow/my-presubmit.sh
 This repository (istio/test-infra) also provides Prow jobs.
 
 - `test-infra-presubmit` - run the linting and testing
+
+### Manually Trigger a Prow Job
+
+
+> **Never do this unless necessary AND you are authorized by istio EngProd team (istio-engprod@google.com)**
```

Comment:
Please remove the istio-engprod as this is google internal, and use the slack channel instead.